cdc/puller/ddl_puller.go

```diff
@@ -42,6 +49,154 @@ const (
 	DDLPullerTableName = "DDL_PULLER"
 )
 
+// DDLJobPuller is used to pull ddl job from TiKV.
+type DDLJobPuller interface {
+	Run(ctx context.Context) error
+	Output() <-chan *model.DDLJobEntry
```

Comment:
Please comment on the timing of the closure of the returned channel, and how it is related to any error conditions that might oocur.